jobserver/releases.py

```diff
@@ -60,29 +58,35 @@ def handle_release(workspace, backend, backend_user, release_hash, upload):
             backend=backend,
             backend_user=backend_user,
             upload_dir=upload_dir,
-            files=[str(f) for f in files],
+            files=files,
         )
 
     except Exception:
         # something went wrong, clean up files, they will need to reupload
-        shutil.rmtree(actual_dir, ignore_errors=True)
+        if actual_dir:
+            shutil.rmtree(actual_dir, ignore_errors=True)
```

Comment:
Doesn't the ignore_errors kwarg being set to true mean we don't need the if statement here?

Reply:
It does, will remove